Dockerfile_intel

```diff
@@ -45,7 +45,7 @@ RUN cargo build --profile release-opt --frozen
 
 # Text Generation Inference base image for Intel
 
-FROM intel/intel-extension-for-pytorch:2.3.110-xpu AS xpu
+FROM intel/oneapi-basekit:2024.2.1-0-devel-ubuntu22.04 AS xpu
```

Comment:
Old base was 3.5GB, oneAPI based is 4GB.

But latest `intel-extension-for-pytorch:2.5.10-xpu]` would be ~5GB, so that does not seem so bad...